tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -239,3 +252,25 @@ Verify Errors In Jupyterhub Logs
         ...    container=${pod['spec']['containers'][0]['name']}
         Should Not Contain    ${logs}    ModuleNotFoundError: No module named 'distutils.util'
     END
+
+Verify Grafana Can Obtain Data From Prometheus Datasource
+    [Documentation]    Verifies Grafana can obtain data from Prometheus datasource
+    Login To Grafana
+    Click Element    xpath=/html/body/grafana-app/sidemenu/div[2]/div[3]/a/span
+    Wait Until Page Contains Element    xpath=//div[contains(@class,'grafana-select-value')]
+    Input Text    xpath=//div[contains(@class,'grafana-select-value')]//input    text=Monitoring
```

Comment:
Lines 260-263 can be a new keyword in Grafana.resource (see comment below)
```
Select Datasource
[Parameters]  ${name}
```

Reply:
Updated.